test/e2e/apimachinery/chunking.go

```diff
@@ -122,6 +122,29 @@ var _ = SIGDescribe("Servers with support for API chunking", func() {
 		gomega.Expect(list.Items).To(gomega.HaveLen(numberOfTotalResources))
 	})
 
+	ginkgo.It("should chunk lists of Pods", func() {
+		ns := f.Namespace.Name
+		c := f.ClientSet
+		client := c.CoreV1().PodTemplates(ns)
+		loopCount := int64(0)
+		for {
+			opts := metav1.ListOptions{}
+			opts.Limit = int64(rand.Int31n(numberOfTotalResources/10) + 1)
```

Comment:
what if there is only 1? 0?

Reply:
`loopCount` is the number of iterations that the `for {` loop has gone through